skorch/callbacks/training.py

```diff
@@ -385,6 +394,12 @@ def __init__(
         self.misses_ = 0
         self.dynamic_threshold_ = None
         self.sink = sink
+        self.load_best = load_best
+
+    def __getstate__(self):
+        # Avoids to save the module_ weights twice when pickling
+        self.best_model_weights_ = None
+        return self.__dict__
```

Comment:
```suggestion
        state = super().__getstate__()
        state['best_model_weights_'] = None
        return state
```

Just to be safe, let's call `super()` here.